scipy/spatial/transform/rotation.py

```diff
@@ -1461,6 +1462,47 @@ def magnitude(self):
         else:
             return angles
 
+    def fundamental(self, group):
+        """Get the fundamental zone representation of the rotation(s).
+
+        Parameters
+        ----------
+        group : `Rotation` instance
+            Object containing the elements of the rotation group.
+
+        Returns
+        -------
+        fundamental : `Rotation` instance
+            Object containing fundamental zone representations of the rotations
+            in the current instance.
+
+        Examples
+        --------
+        >>> from scipy.spatial.transform import Rotation as R
+        >>> from scipy.spatial.transform import RotationGroup
+        >>> r = R.from_quat([0.5, 0.5, 0.5, 0.5])
+        >>> r.fundamental(RotationGroup.octahedral).as_quat()
+        array([0., 0., 0., 1.])
+
+        >>> r.fundamental(RotationGroup.cyclic(2)).as_quat()
+        array([0.5, 0.5, 0.5, 0.5])
+        """
+
+        quat = self._quat.reshape((len(self), 4))
+        qsym = group.as_quat().reshape((len(group), 4))
+
+        dots = np.outer(quat[:, 3], qsym[:, 3])
```

Comment:
As far as I understand these `dots` are cosine of the angle between rotations (just for myself).

Reply:
Yes.  I have renamed them for to `cosines` for clarity.